PWG/EMCAL/EMCALtasks/AliEmcalMCTrackSelector.cxx

```diff
@@ -38,14 +38,17 @@
 #include "AliMCParticle.h"
 #include "AliStack.h"
 #include "AliNamedArrayI.h"
+#include "AliVEventHandler.h"
 
 #include "AliLog.h"
 
 ClassImp(AliEmcalMCTrackSelector)
 
-AliEmcalMCTrackSelector::AliEmcalMCTrackSelector() : 
+AliEmcalMCTrackSelector::AliEmcalMCTrackSelector() :
   AliAnalysisTaskSE("AliEmcalMCTrackSelector"),
   fParticlesOutName("MCParticlesSelected"),
+  fTrackContainerName("usedefault"),
```

Comment:
You cannot use the track container name "usedefault" like this, there is no such object attached to the event. We use "usedefault" in order to tell the code  - existing EMCAL containers - to auto-determine the default container name for certain objects, however those have helper functions to build the default name which is then provided to `FindListObject`. You pass "usedefault" directly to `FindListObject`. I think using "usedefault" is a good idea, but then you need to translate it later into the actual default container name.